lib/io/fileutils/fileutils.go

```diff
@@ -172,7 +177,12 @@ func WriteToFile(filepath string, data []byte) error {
 }
 
 func ReadFile(filename string) ([]byte, error) {
-	return ioutil.ReadFile(filename) // nosemgrep
+	home := os.Getenv("HOME")
+	path := filename
+	if strings.Index(path, "~") == 0 {
+		path = strings.Replace(path, "~", home, 1)
+	}
+	return os.ReadFile(path)
```

Comment:
what if tilde is given but HOME env var is not set, should we give error or its ok to keep HOME empty ?

Reply:
sure